atlauncher.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash -e
+# Credits to github/nickavem for the script, Doomsdayrs for edits
+
+ln -sfn ~/.var/app/com.atlauncher.ATLauncher/cache/ ~/.var/app/com.atlauncher.ATLauncher/data/cache
+ln -sfn ~/.var/app/com.atlauncher.ATLauncher/config ~/.var/app/com.atlauncher.ATLauncher/data/configs
+
+DIR=${CUSTOM_DIR:-".var/app/com.atlauncher.ATLauncher/data/"}
```

Comment:
I'm not sure if I follow what this is for?

Reply:
Cache and Config file redirection to the proper folders

Reply:
The second part with DIR= is for a custom work dir

Reply:
It's still unclear to me who would use it, and how they are supposed to discover this without being documented anywhere.

Reply:
Solved with adding documentation in a86551e